shell/components/formatter/LiveDuration.vue

```diff
@@ -0,0 +1,113 @@
+<script>
+import day from 'dayjs';
+
+/**
+ * return { diff: number; label: string }
+ *
+ * diff:  update frequency in seconds
+ * label: content of the cell's column
+ */
+function elapsedTime(seconds) {
```

Comment:
Am I right in thinking that you've written this instead of using `diffFrom` (used to format the age column) because `diffFrom` does a lot of rounding? I think it makes sense to be more precise in this column, I just want to verify that I've understood the code correctly. The function seems general enough that it should probably live in `/shell/utils/time` though. Otherwise the PR looks good, and seems to work correctly.